src/Widgets/HyperTextView.vala

```diff
@@ -66,24 +66,20 @@
             focus_out_event.connect (on_focus_out_event);
 
             /**
-             * Binding key_press/key_release signals to toplevel window
-             * if possible enables us to detect when the Control key
-             * is pressed even when HyperTextView is not focused.
-             */
-
-            Gtk.Window? toplevel_window = null;
-            foreach (unowned var window in Gtk.Window.list_toplevels ()) {
-                if (window.get_parent_window () == null) {
-                    toplevel_window = window;
-                    break;
+             * Binding key_press/key_release signals to all toplevel
+             * windows possible enables us to detect when the Control
+             * key is pressed even when HyperTextView is not focused.
+            */
+            var toplevel_windows = Gtk.Window.list_toplevels ();
+
+            if (toplevel_windows.length () != 0) {
+                foreach (unowned var toplevel_window in Gtk.Window.list_toplevels ()) {
```

Comment:
Looks like this should either use `toplevel_windows` or we should get rid of that variable altogether.

Reply:
Uuups - yeah, right! 🤭